astro/src/content/docs/apis/_user-registration-combined-request-body.mdx

```diff
@@ -110,7 +110,7 @@ This request requires that you specify both the User object and the User Registr
   <APIField name="user.password" type="String" optional>
     The User's plain text password. This password will be hashed and the provided value will never be stored and cannot be retrieved.
 
-    This field is optional only if `sendSetPasswordEmail` is set to `true`. By default `sendSetPasswordEmail` is `false`, and then this field will be required.
+    If `tenant.passwordEnabled` is true, this field is required unless `sendSetPasswordEmail` or `sendSetPasswordPhone` is set to `true`. By default, `sendSetPasswordEmail` and `sendSetPasswordPhone` are `false`.
```

Comment:
```suggestion
    This field is required unless `tenant.passwordEnabled` is `false`, or either `sendSetPasswordEmail` or `sendSetPasswordPhone` is `true`. Note that by default, `sendSetPasswordEmail` and `sendSetPasswordPhone` are `false`.
```

Reply:
@andrewpai forgot to rebase this after the send set pass API changes. Sorry about that. Did that and took a stab at incorporating your language in there. wasn't sure whether to say "something besides `doNotSend`", like I did, or use "not", but didn't want _too_ boolean of a sentence. Open to suggestions.